README.md

```diff
@@ -14,11 +14,7 @@
 - [Using from the command-line interface (CLI)](#using-from-the-command-line-interface-cli)
   * [Install the CLI](#install-the-cli)
   * [Update the CLI](#update-the-cli)
-  * [CLI usage](#cli-usage)
-  * [Global templates installed with yarn or npm](#global-templates-installed-with-yarn-or-npm)
-  * [CLI usage examples](#cli-usage-examples)
-  * [CLI usage with Docker](#cli-usage-with-docker)
-  * [CLI usage with npx instead of npm](#cli-usage-with-npx-instead-of-npm)
+  * [CLI usage](usage.md)
```

Comment:
you cannot update TOC manually as we have automation that will regenerate anyway

just make sure we have `CLI usage` section below, and it is ok if it just contains one sentence that links to another document

Reply:
Done